### PR TITLE
Update for recent CodeMirror change for Sprint 36

### PR DESCRIPTION
**Pull request by redmunds**
This is for #163 

This fix uses some educated guesses derived by looking at [this commit](https://github.com/adobe/brackets/commit/76a5e7de8a6e43b894187a46c71ef1f9d1fc3c36#diff-205888d21d20aa2319d86062c2e86080) by @njx 

cc @RaymondLim

### CODE REVIEW

File: cssModeSupport.js [hunk 1]
```diff
@@ -43,12 +43,11 @@ define(function (require, exports, module) {
...
     
     function inRuleBody(t) {
-        var stateStack = t.state.stack || t.state.localState.stack;
+        var context = t.state.context || t.state.localState.context;
         
         return t.className !== "property" &&
             t.className !== "property error" &&
-            stateStack.length > 0 &&
-            stateStack[stateStack.length - 1] === "propertyValue";
+            context.type === "propertyValue";
```
Comment (RaymondLim) [inline cssModeSupport.js]: @redmunds I think it should compare against "prop".

Update: CM css tokenizer no longer using "propertyValue".

**Comment by RaymondLim** [comment]
<del>We also need to either update lessModeSupport.js or remove it and combine the logic with cssModeSupport. </del>

Never mind. I didn't notice your change to lessModeSupport.js file earlier. How about SCSS support? I think we can combine it with CSS Mode. That's what I did in Brackets.

**Comment by RaymondLim** [comment]
Looks good on code changes. I'll merge after I verify it working for scss and less.

**Comment by RaymondLim** [comment]
I'm seeing an issue -- when cursor is in a font name that is not wrapped inside quotes, then hint list is not mapping to the prefix.

**Comment by redmunds** [comment]
Looks good. Merging.

**Pull request merged**